cmake/FindLibUSB.cmake

```diff
@@ -0,0 +1,136 @@
+# This is a modified version of the file written by Hedrik Sattler,
+# from the OpenOBEX project (licensed GPLv2/LGPL).  (If this is not correct,
```

Comment:
We are MIT licensed here so while we could include this anyone using lib-seek in commercial software couldn't. I'm going to take a hack at writing a version which we can put under our license.

Reply:
It's dual licensed with LGPL, that should be ok to keep for commercial, especially since it's not code that will be linked, but then again IANAL.